general_rules/Organization.tex

```diff
@@ -23,31 +23,17 @@ \subsection{Stage System}\label{rule:stages}
 Note that the same task can be performed multiple times during \SONE{} and \STWO{} (see~\refsec{rule:score_system}).
 In case of having no considerable score deviation between a team advancing to the next stage and a team dropping out, the TC may announce additional teams advancing to the next stage.
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection{Thematic Scenarios}
-\label{rule:themes}
-
-Each stage includes a set of tasks that are grouped in two thematic scenarios:
-\begin{enumerate}
-	\item The \iterm{Housekeeper} scenario features tasks related to cleaning, organization, and maintenance of a household.
-	\item The \iterm{Party Host} scenario focuses on providing general assistance during a party by attending to the needs of the guests.
-\end{enumerate}
```

Comment:
I think it would be good to keep the thematic scenarios as these are two different contexts in which the robot is employed. Otherwise it might become unclear what the goal of the robot is.

Reply:
I think leaving in scenarios when they have no impact on scheduling or rules is confusing for new readers